nautobot/extras/navigation.py

```diff
@@ -318,4 +318,23 @@
             ),
         ),
     ),
+    NavMenuTab(
+        name="Plugins",
+        weight=5000,
+        groups=(
+            NavMenuGroup(
+                name="General",
+                weight=100,
+                items=(
+                    NavMenuItem(
+                        link="plugins:plugins_list",
+                        name="Installed Plugins",
+                        weight=100,
+                        permissions=[],
```

Comment:
Sorry I'm late to the party,
I think it would be good to have a permission associated with this menu item and this view to allow Nautobot Admin to limit who can access this view

Reply:
Currently the view is available to all authenticated users. Since there isn't an actual data model underpinning it, it's a bit tricky to use the object-based permissions framework to restrict the view more narrowly than that. I'm open to suggestions though.